net/openfortivpn/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        adrienverge openfortivpn 1.7.1 v
+
+categories          net
+platforms           darwin
+license             GPL-3
+
+maintainers         {gmail.com:millerdev @millerdev} openmaintainer
+description         Client for PPP+SSL VPN tunnel services.
+long_description    openfortivpn is a client for PPP+SSL VPN tunnel services. \
+                    It spawns a pppd process and operates the communication \
+                    between the gateway and this process. It is compatible \
+                    with Fortinet VPNs.
+
+depends_build       port:pkgconfig
+depends_lib         port:openssl
+
+use_autoreconf      yes
+configure.pkg_config    ${prefix}/bin/pkg-config
```

Comment:
Why did you add the `configure.pkg_config` line? I successfully built this port without it.